migrations/sql/V2024.12.10.15.15__create_users_table.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE "user"
+(
+    sub               VARCHAR PRIMARY KEY,
+    email             VARCHAR                                NOT NULL,
+    given_name        VARCHAR                                NOT NULL,
+    family_name       VARCHAR                                NOT NULL,
+    display_name      VARCHAR                                NOT NULL,
+    idir_username     VARCHAR                                NOT NULL,
+    identity_provider VARCHAR                                NOT NULL,
+    idir_user_guid    VARCHAR                                NOT NULL,
+    last_logged_in    TIMESTAMPTZ,
+    create_user       VARCHAR(255)                           NOT NULL,
+    create_timestamp  timestamp with time zone DEFAULT now() NOT NULL,
```

Comment:
Just curious what the difference is between "TIMESTAMPTZ" and "timestamp with time zone" (or are they the same but just copied and pasted from different places? Because that is 100% perfectly fine)

Reply:
Just two ways of referring to the same data type.  It resolves to the same thing.